gobblin-api/src/main/java/org/apache/gobblin/configuration/ConfigurationKeys.java

```diff
@@ -1014,7 +1014,7 @@ public class ConfigurationKeys {
   public static final String DATASET_BASE_OUTPUT_PATH_KEY = "gobblin.flow.dataset.baseOutputPath";
   public static final String DATASET_COMBINE_KEY = "gobblin.flow.dataset.combine";
   public static final String WHITELISTED_EDGE_IDS = "gobblin.flow.whitelistedEdgeIds";
-
+  public static final String GOBBLIN_FLOW_ISADHOC = "gobblin.flow.isAdhoc";
```

Comment:
should we debug a separate namespace for these, which are meant to be internal-only, not to be supplied by a user?  e.g. `gobblin.internal.flow.isAdhoc`